tests/robustness/scenarios.go

```diff
@@ -106,8 +106,17 @@ func exploratoryScenarios(_ *testing.T) []testScenario {
 		clusterOfSize1Options = append(clusterOfSize1Options, e2e.WithClusterSize(1))
 		// Add LazyFS only for traffic with lower QPS as it uses a lot of CPU lowering minimal QPS.
 		if enableLazyFS && tp.Profile.MinimalQPS <= 100 {
-			clusterOfSize1Options = append(clusterOfSize1Options, e2e.WithLazyFSEnabled(true))
-			name = filepath.Join(name, "LazyFS")
+			// Set CompactionBatchLimit to default when LazyFS is enabled, because frequent compaction uses a lot of CPU too.
```

Comment:
I didn't meant WithCompactionBatchLimit, but executing compaction in traffic. Still let's see if this is enough.

Reply:
I was thinking of something more like https://github.com/etcd-io/etcd/pull/18181